lib/wings/active_fedora_converter.rb

```diff
@@ -163,13 +163,21 @@ def result
       def convert_members(af_object)
         return unless resource.respond_to?(:member_ids) && resource.member_ids
         # TODO: It would be better to find a way to add the members without resuming all the member AF objects
-        resource.member_ids.each { |valkyrie_id| af_object.ordered_members << ActiveFedora::Base.find(valkyrie_id.id) }
+        ordered_members = []
+        resource.member_ids.each do |valkyrie_id|
+          ordered_members << ActiveFedora::Base.find(valkyrie_id.id)
+        end
+        af_object.ordered_members = ordered_members
```

Comment:
This line actually persists the members in fedora. The safe in the persister on the parent completes the transaction and triggers a reindexing of everything. Ditto with line 180 below for collections. I believe that this still needs some re-work.